### DISCUSSION TITLE
Unable to renew my status because of the special student ID

### DISCUSSION THREAD

**Discussion by rexlow** [community/community#49576] [GitHub Education]
Our university issued a bank-account linked student ID and my name is printed at the back, while the expiry date and mugshot is printed on the front.

Because of this reason, GitHub keep insisting that the student ID is not valid, although I have both pictures in a picture. 

Is there anyway or anyone I can contact with?

**Comment by samus-aran** [comment]
Hi @rexlow 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
We do accept Student ID that may not contain all details, so we would recommend you acquire additional evidence and take a photo along side the Student ID.

Let us know how you get on 👍
